### ISSUE TITLE
[INFO] using 'STDIN' for parsing targets list

### ISSUE THREAD

**Issue by chevalier6290** [sqlmapproject/sqlmap#4720]
The program cannot run, stops for a long time at [INFO] using 'STDIN' for parsing targets list.

![image](https://user-images.githubusercontent.com/81340702/122939711-1117d480-d39e-11eb-8892-0dfe981c8e1f.png)

**Comment by stamparm** [comment]
something is totally messed with your runtime environment. it appears that the `stdin` is redirected in some strange way (prior to the program run <- i can see that you had colored terminal output prior to the run of `sqlmap`)

**Comment by h1pmnh** [comment]
For anyone else who ends up at this issue from a Github search, I found that this behavior changed between sqlmap 1.4 and 1.5 at least on Windows (git bash). sqlmap requires a full terminal as suggested in the response by @stamparm and this error can be bypassed on Windows by running it with a `winpty` prefix, e.g. `winpty python -u sqlmap.py ...`

Note for me, I saw this behavior after upgrading from sqlmap 1.4.9 to 1.5.10. Hope this helps someone else who ends up at this Github issue :)
